srcpkgs/gdb/template

```diff
@@ -21,6 +22,13 @@ distfiles="${GNU_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
 checksum=f82f1eceeec14a3afa2de8d9b0d3c91d5a3820e23e0a01bbb70ef9f0276b62c0
 python_version=3
 patch_args="-Np1"
+ignore_elf_files="
+ /usr/share/gdb/guile/gdb/support.go
+ /usr/share/gdb/guile/gdb/experimental.go
+ /usr/share/gdb/guile/gdb/iterator.go
+ /usr/share/gdb/guile/gdb/types.go
+ /usr/share/gdb/guile/gdb/printing.go
+ /usr/share/gdb/guile/gdb.go"
```

Comment:
This should not be necessary, see #21288

Reply:
This seems to be necessary.
in `11-pkglint-elf-in-usrshare.sh`:
```sh
        case "${mime}" in
            application/x-sharedlib*|\
             application/x-pie-executable*|\
             application/x-executable*)
                if [[ ${ignore_elf_files} != *"${file}"* ]]; then
                    matches+=" ${file}"
                fi
                ;;
```
Sample *.go file:
```sh
$ readelf -h texinfo.go | grep Type
  Type:                              DYN (Shared object file)
$ file --mime-type texinfo.go 
texinfo.go: application/x-sharedlib
```

Reply:
You are right. It makes sense to make the same changes in 11 too. Probably will do it later.